x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/logs/logs_tab_content.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import type { Filter } from '@kbn/es-query';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
+import { SnapshotNode } from '../../../../../../../common/http_api';
+import { LogStream } from '../../../../../../components/log_stream';
+import { useHostsViewContext } from '../../../hooks/use_hosts_view';
+import { useUnifiedSearchContext } from '../../../hooks/use_unified_search';
+import { useLogsSearchUrlState } from '../../../hooks/use_logs_search_url_state';
+import { LogsLinkToStream } from './logs_link_to_stream';
+import { LogsSearchBar } from './logs_search_bar';
+
+export const LogsTabContent = () => {
+  const [filterQuery] = useLogsSearchUrlState();
+  const { getDateRangeAsTimestamp } = useUnifiedSearchContext();
+  const { from, to } = getDateRangeAsTimestamp();
+  const { hostNodes, loading } = useHostsViewContext();
+
+  const hostsFilterQuery = useMemo(() => createHostsFilter(hostNodes), [hostNodes]);
+
+  const logsLinkToStreamQuery = useMemo(() => {
+    const hostsFilterQueryParam = createHostsFilterQueryParam(hostNodes);
+    return `${filterQuery.query ? filterQuery.query + ' and ' : ''}${hostsFilterQueryParam}`;
+  }, [filterQuery.query, hostNodes]);
+
+  if (loading)
+    return (
+      <EuiFlexGroup style={{ height: 300 }} alignItems="center" justifyContent="spaceAround">
+        <EuiFlexItem grow={false}>
+          <EuiLoadingSpinner size="xl" />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="m" data-test-subj="hostsView-logs">
+      <EuiFlexGroup gutterSize="m" alignItems="center" responsive={false}>
+        <EuiFlexItem>
+          <LogsSearchBar />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <LogsLinkToStream startTimestamp={from} endTimestamp={to} query={logsLinkToStreamQuery} />
```

Comment:
This is probably a weird scenario since the button opens the Logs UI in the **same** browser tab, meaning that if the user returns to the Hosts view via the browser's back button, the whole thing will be reloaded again. But just out of curiosity, since opening a new tab is a common practice, I did the steps below:

- Open the Logs tab, with the date range set to 15m-now
- Open the Logs UI in a **new tab** (should show something like 12:00:00 -> 12:15:00
- Go back to the Hosts view browser tab, switch to the Metrics tab, and wait 1 minute
- Reopen the Logs Tab and open the Logs UI in a new tab.

The date range in the Logs UI will now be 12:01:00 -> 12:16:00. 

Question: Is this the expected behavior?

Reply:
I guess that the time should always be the same when you open in a new tab as long as the time filter didn't change.
which in your test case should be `12:00:00 -> 12:15:00` and not `12:01:00 -> 12:16:00.`

I believe this has to do with the change in this [comment](https://github.com/elastic/kibana/pull/152995#discussion_r1146130896).